http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeonardoModel.scala

```diff
@@ -106,19 +106,37 @@ case class AuditInfo(creator: WorkbenchEmail,
                      dateAccessed: Instant,
                      kernelFoundBusyDate: Option[Instant])
 
-sealed trait ClusterTool extends EnumEntry with Serializable with Product
-object ClusterTool extends Enum[ClusterTool] {
+sealed trait ClusterContainerServiceType extends EnumEntry with Serializable with Product {
+  def imageType: ClusterImageType
+}
+object ClusterContainerServiceType extends Enum[ClusterContainerServiceType] {
```

Comment:
I'm curious why we need both `ClusterContainerServiceType` and `ClusterImageType` enums -- they seem duplicative of each other. When do we use one v. the other?

Reply:
the main difference is `ClusterImageType` can include `customDataproc`, vs `ClusterContainerServiceType` doesn't include that